cli/compose/convert/volume.go

```diff
@@ -135,6 +155,8 @@ func convertVolumeToMount(
 		return handleBindToMount(volume)
 	case "tmpfs":
 		return handleTmpfsToMount(volume)
+	case "npipe":
+		return handleNpipeToMount(volume)
 	}
 	return mount.Mount{}, errors.New("volume type must be volume, bind, or tmpfs")
```

Comment:
Error message should now include npipe as a valid mount type.

Reply:
Good point. I included it to commit now.